aggsender/block_notifier_polling.go

```diff
@@ -170,7 +170,7 @@ func (b *BlockNotifierPolling) step(ctx context.Context,
 	}
 	newState := previousState.incommingNewBlock(currentBlock)
```

Comment:
Nitpick :slightly_smiling_face: 
```suggestion
	newState := previousState.incomingNewBlock(currentBlock)
```